File: source/DataTransferService/DTS/DTLib/log/collection/Event.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

using MongoDB.Driver;
using MongoDB.Bson;
namespace DTLib.log.collection
{
    public class Event : Base
    {
        public ObjectId id { get; set; }
        public string machine { get; set; }
        public string eventType { get; set; }
        public string eventID { get; set; }
        public string eventGroup { get; set; }
        public string unitID { get; set; }
        public string slotID { get; set; }
        public string date { get; set; }
        public string time { get; set; }
        public string message { get; set; }
        public DateTime datetime { get; set; }
    }
}

File: source/DataProvider/api/models/Event.js
/**
 * Event.js
 *
 * @description :: TODO: You might write a short summary of how this model works and what it represents here.
 * @docs        :: http://sailsjs.org/documentation/concepts/models-and-orm/models
 */

module.exports = {
  tableName:'Event',
  attributes: {
    deleted: 'bool',
    createtime: 'datetime',
    creator: 'string',
    modifietime: 'datetime',
    revisor: 'string',
    machine: 'string',
    eventType: 'string',
    eventID: 'string',
    eventGroup: 'string',
    unitID: 'string',
    slotID: 'string',
    date: 'date',
    time: 'datetime',
    message: 'string',
    datetime:'datetime'
  }
};

